content-resources/src/main/java/flyway/oskari/V1_32_1__populate_capabilities_cache.java

```diff
@@ -78,11 +69,12 @@ public void migrate(Connection connection) throws SQLException {
     }
 
     private void insertCaps(Connection conn, OskariLayerCapabilities caps) throws SQLException {
-        final String sql = "INSERT INTO oskari_capabilities_cache (layertype, url, data) VALUES(?,?,?)";
+        final String sql = "INSERT INTO oskari_capabilities_cache (layertype, url, data, version) VALUES(?,?,?,?)";
```

Comment:
Version column is added after this migration: https://github.com/oskariorg/oskari-server/blob/master/content-resources/src/main/resources/flyway/oskari/V1_36_4__add_version_column_to_oskari_capabilities_cache.sql so it isn't part of the table at this point. There's all kinds of trouble to be found when updating old migrations :)